internal/db/schema/migrations/oss/postgres/99/01_server_worker_release_version.up.sql

```diff
@@ -0,0 +1,53 @@
+begin;
+
+  alter table server_worker
+    add column release_version text;
+
+drop view server_worker_aggregate;
+
+-- Updates view created in 34/04_views.up.sql
+create view server_worker_aggregate as
+with worker_config_tags(worker_id, source, tags) as (
+  select
+    ct.worker_id,
+    ct.source,
+    -- keys and tags can be any lowercase printable character so use uppercase characters as delimitors.
+    string_agg(distinct concat_ws('Y', ct.key, ct.value), 'Z') as tags
+  from server_worker_tag ct
+  group by ct.worker_id, ct.source
+),
+     connection_count (worker_id, count) as (
+       select
+         worker_id,
+         count(1) as count
+       from session_connection
+       where closed_reason is null
+       group by worker_id
+     )
+select
+  w.public_id,
+  w.scope_id,
+  w.description,
+  w.name,
+  w.address,
+  w.create_time,
+  w.update_time,
+  w.version,
+  w.last_status_time,
+  w.type,
+  w.release_version,
+  cc.count as active_connection_count,
+  -- keys and tags can be any lowercase printable character so use uppercase characters as delimitors.
+  wt.tags as api_tags,
+  ct.tags as worker_config_tags
+from server_worker w
```

Comment:
The formatting in this section varies from the version in 34/04. It doesn't seem intentional, so calling it out.

Reply:
Thanks for catching that!